### PR TITLE
Improve setup instructions and use local node binaries.

### PR DESCRIPTION
**Pull request by schmave**

### CODE REVIEW

**Comment by schmave** [comment]
Hi there! I work on the software that Philly Free School uses for managing some student records ("DemSchoolTools"). I want to see if it will be relatively easy to integrate Overseer into DemSchoolTools so that it can be used by the other schools that use DST. I'm thinking at this point that I'll modify Overseer to read student names from DemSchoolTools, and then run Overseer in an iframe or something so that the web stuff doesn't have to be changed much. Curious if you have any thoughts.

I got setup without too much trouble, but did run into a couple of stumbling blocks, which I have reported about here.

File: package.json [hunk 1]
```diff
@@ -39,10 +39,9 @@
     "babelify": "6.3.0",
     "browserify": "11.0.1",
     "concurrently": "2.0.0",
-    "jest-cli": "0.5.0",
     "reactify": "1.1.1",
     "uglifyify": "3.0.1",
     "watchify": "3.4.0",
-    "shrinkwrap": "6.0.1"
+    "npm-shrinkwrap": "6.0.1"
```
Comment (schmave) [inline package.json:45]: I see a package called "shrinkwrap" that has version numbers starting with "0.", so I guess you meant "npm-shrinkwrap" here? Not sure...

**Comment by steveshogren** [comment]
Hey, thanks for the PR!

Are you associated with one of the other free schools? I started this for the Philly Free School. It's obviously open and free for you to host and use at your school. If you'd like to use it for your school without hosting it I've got an AWS instance up and running and can easily setup a new account for your school.

If you have any questions I'm happy to help!

File: project.clj [hunk 2]
```diff
@@ -44,7 +44,8 @@
             ;;[refactor-nrepl "2.0.0-SNAPSHOT"]
             [lein-ring "0.9.7"]
             [migratus-lein "0.2.6"]
-            [environ/environ.lein "0.2.1"]]
+            [environ/environ.lein "0.2.1"]
+            [lein-environ "1.1.0"]]
```
Comment (schmave) [inline project.clj:48]: This plugin generates a `.lein-env` file containing the environment variables from the project + profile. I couldn't get the database-url set in the environment without adding it. Is there another way?

File: package.json (hunk 1 above)
Comment (steveshogren) [reply package.json:45]: Haha, JS libs are here one day, gone the next... If it works, great!

**Comment by steveshogren** [comment]
Hey sounds great. I'm floored someone else is out there making software for these schools who ALSO can get a clojure site up and running. 

Honestly DemSchoolTools sounds way more sophisticated, so I'm not sure why you'd want overseer integrated, but I'm definitely happy to collaborate! 

Do you host it and charge for hosting? Or install locally? 

I'll get DST setup and poke around with it. I could likely modify overseer to read from a different schema for student records and classes.

**Comment by schmave** [comment]
Thanks for the prompt review! I'd like to talk more--could you send me an email at schmave@gmail.com?

**Pull request merged**